supported_hardware.md

```diff
@@ -0,0 +1,35 @@
+# Supported Hardware for the Framework laptop!
```

Comment:
This should be a separate pull request from the Linux Tips, please separate it out so it can be reviewed on its own.